app/controllers/katello/api/v2/host_subscriptions_controller.rb

```diff
@@ -35,6 +37,42 @@ def events
       respond_for_index :collection => collection
     end
 
+    api :POST, "/hosts/subscriptions/register", N_("Simulate a host subscription-manager registration")
```

Comment:
Plenty, but they haven't changed. This just looks like a copy/paste of the RHSM API so that users feel like they are using a "real" API to do what RHSM does instead of just simulating RHSM since that is our client tool. When this code or the RHSM code changes, what is to prevent them from becoming inconsistent with each other functionality wise?